nucliadb_protos/writer.proto

```diff
@@ -444,9 +445,29 @@ message DelVectorSetResponse {
 }
 
 
+message NewKnowledgeBoxV2Request {
+    string kbid = 1;
+    string kb_slug = 2;
+    string title = 3;
+    string description = 4;
+
+    message VectorSet {
+        string vectorset_id = 1;
+        utils.VectorSimilarity similarity = 2;
+        uint32 vector_dimension = 3;
+        repeated uint32 matryoshka_dimensions = 4;
+    }
+    repeated VectorSet vectorsets = 5;
```

Comment:
Why not reuse the existing message `VectorSetConfig`?

Reply:
Because there's a bunch of params that won't be populated (normalize and vector type), but I can change it if you consider it cleaner

Reply:
Ah, I misread that message. Indeed, it makes sense to have it be a separate thing. :+1:  Thanks!